components/board/digital_interrupts.go

```diff
@@ -123,17 +129,17 @@ func (i *BasicDigitalInterrupt) Value(ctx context.Context, extra map[string]inte
 }
 
 // Ticks is really just for testing.
-func (i *BasicDigitalInterrupt) Ticks(ctx context.Context, num int, now uint64) error {
+func (i *BasicDigitalInterrupt) Ticks(ctx context.Context, num int, now uint32) error {
 	for x := 0; x < num; x++ {
-		if err := i.Tick(ctx, true, now+uint64(x)); err != nil {
+		if err := i.Tick(ctx, true, now+uint32(x)); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
 // Tick records an interrupt and notifies any interested callbacks.
-func (i *BasicDigitalInterrupt) Tick(ctx context.Context, high bool, not uint64) error {
```

Comment:
:flushed: It's a good thing this line is changing: a variable named `not` is confusing, and I suspect it was a typo of `now`.